src/Tests.js

```diff
@@ -52,14 +52,14 @@ function* readAllCharacteristics(device: Device): Generator<*, boolean, *> {
 
         for (const descriptor of descriptors) {
           yield put(log('* Found descriptor: ' + descriptor.uuid));
-          const d: Descriptor = yield call([descriptor, descriptor.read]);
-          yield put(log('Descriptor value: ' + (d.value || 'null')));
-          if (d.uuid === '00002902-0000-1000-8000-00805f9b34fb') {
-            yield put(log('Skipping CCC'));
-            continue;
-          }
+        //   const d: Descriptor = yield call([descriptor, descriptor.read]);
```

Comment:
why is it commented out instead of removed?

Reply:
Read/write operation on Descriptors and Characteristics are not yet implemented, so the test failed. They'll be gradually reintroduced as we roll out next features.